qemu/tests/cfg/numa_prealloc_threads.cfg

```diff
@@ -0,0 +1,20 @@
+- numa_prealloc_threads:
+    no RHEL.6 RHEL.7 RHEL.8
+    no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8
+    required_qemu = [7.2,)
+    type = numa_prealloc_threads
+    virt_test_type = qemu
+    login_timeout = 1000
+    qemu_command_prefix = "taskset -c 0 "
+    vm_thread_contexts = "tc1"
+    smp_fixed = 8
+    vcpu_maxcpus = ${smp_fixed}
+    vm_thread_context_cpu-affinity = "1-7"
+    sandbox_error_message = "Setting CPU affinity failed: Operation not permitted"
```

Comment:
It is for the default `qemu_sandbox = on` only

Reply:
Updated, thanks!